airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -136,14 +134,14 @@ def read(
 
     @property
     def per_stream_state_enabled(self):
-        return False  # While CDK per-stream is in active development we should keep this off
+        return True  # While CDK per-stream is in active development we should keep this off
```

Comment:
will switch this back before merge, but while i test/commit changes, it easier to leave this on